README.md

```
@@ -232,3 +232,21 @@ Have questions or feedback? We're here to help:
+
+## 🧠 Permanent Brain Integration (LangGraph)
+
+### Architecture Flow
+mermaid
+sequenceDiagram
+    participant SG as StateGraph
+    participant CPS as MemantoCheckpointSaver
+    participant SMC as SemanticMemoryController
+    participant SDK as Memanto SDK
+    
+    SG->>SMC: Process input for semantic value
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)